HW_005/task-02.py

```diff
@@ -0,0 +1,19 @@
+'''
+Задача 28: 
+Напишите рекурсивную функцию sum(a, b), возвращающую сумму 
+двух целых неотрицательных чисел. Из всех арифметических операций 
+допускаются только +1 и -1. Также нельзя использовать циклы.
+*Пример:*
+2 + 2 = 4 
+'''
+
+slog_1 = int(input('Введите первое слогаемое: '))
+slog_2 = int(input('Введите второе слогаемое: '))
+
+def summa(sl_1, sl_2):
+    if sl_2 == 0:
+        return sl_1
+    else:
+        return summa(sl_1, sl_2 - 1) + 1
```

Comment:
Обычно блок кода `else` в таких случаях не используется.
```suggestion
    return summa(sl_1, sl_2 - 1) + 1
```